includes/class-wc-calypso-bridge-mu-events.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Events that must be registered even without WooCommerce detected.
+ *
+ * @package WC_Calypso_Bridge/Classes
+ * @since   1.0.0
+ * @version 1.0.0
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge Mu Events
+ */
+class WC_Calypso_Bridge_Mu_Events {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_Themes_Setup instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	private function __construct() {
+		add_action( 'woocommerce_admin_newly_installed', array( $this, 'maybe_create_wc_pages' ), 10, 2 );
```

Comment:
I thought we already tried this hook once before - and it seemed to never fire in the context of the ecomm plan?

Reply:
@timmyc We did, but it never got executed. It's a bit hard to explain and it's confusing, but the reason the hook did not get called was because `wc-calypso-bridge` never get a chance to add hooks due to [this line](https://github.com/Automattic/wc-calypso-bridge/blob/master/wc-calypso-bridge.php#L20).

With the change in this PR, the hook should work.